lib/lightning/accounts/user_notifier.ex

```diff
@@ -415,4 +415,54 @@ defmodule Lightning.Accounts.UserNotifier do
 
   defp pluralize_with_s(1, string), do: string
   defp pluralize_with_s(_integer, string), do: "#{string}s"
+
+  @doc """
+  Deliver instructions to confirm a credential transfer.
+  """
+  def deliver_credential_transfer_confirmation_instructions(
+        owner,
+        receiver,
+        credential,
+        token
+      ) do
+    validity_days = Lightning.Config.credential_transfer_token_validity_in_days()
+    validity_text = format_validity_period(validity_days)
+
+    confirmation_url =
+      url(~p"/credentials/transfer/#{credential.id}/#{receiver.id}/#{token}")
+
+    deliver(owner, "Confirm your credential transfer", """
+    Hi #{owner.first_name},
```

Comment:
Nice way to enforce the real person in case the password/login is shared among multiple people.